easybuild/tools/build_log.py

```diff
@@ -299,3 +300,34 @@ def print_warning(message, silent=False):
     """
     if not silent:
         sys.stderr.write("\nWARNING: %s\n\n" % message)
+
+
+def time_str_since(start_time):
+    """
+    Return string representing amount of time that has passed since specified timestamp
+
+    :param start_time: datetime value representing start time
+    :return: string value representing amount of time passed since start_time;
+             format: "[[%d hours, ]%d mins, ]%d sec(s)"
+    """
+    tot_time = datetime.now() - start_time
+    if tot_time.seconds > 0:
+        hours, mins, secs = 0, 0, tot_time.seconds
+        if secs >= 60:
```

Comment:
this can be done better I hope? Take total_seconds, use it as timestamp input and use the buildin formatting functions?

Reply:
fixed, but apparently this doesn't work with Python 2.6... :(

```
AttributeError: 'datetime.timedelta' object has no attribute 'total_seconds'
```